src/templates/components/Hero.template.tsx

```
@@ -358,15 +358,15 @@ export function SplitHero(props: Omit<HeroProps, 'layout'>) {
 export function BackgroundImageHero(
   props: Omit<HeroProps, 'layout' | 'backgroundImage'> & {
     backgroundImage: string;
-  }
+  },
 ) {
   return <Hero {...props} layout="background-image" />;
 }
 
 export function VideoHero(
```

Biome (2.1.2)

[error] Shouldn't redeclare 'VideoHero'. Consider to delete it or rename it.

'VideoHero' is defined here:


(lint/suspicious/noRedeclare)